templates/base/package.json

```diff
@@ -14,7 +14,7 @@
     "storybook:install": "cd storybook && yarn",
     "storybook:run": "cd storybook && yarn storybook",
     "storybook:build": "cd storybook && yarn build-storybook",
-    "generate": "plop --plopfile generators/index.js",
+    "generate": "plop --plopfile generators/index.js && npx prettier --write src/components/",
```

Comment:
1-2 issues here: 

1. NPX is going to install the prettier every-time where as we already have it as dependency. 
2. This command will be slow for whole project if they become really big, We want to run it on the component that is being generated only. 

So this you need to do it by using prettier API.

Reply:
Changed to pretty-quick